docs/README.OLD.md

```diff
@@ -0,0 +1,7 @@
+# DAOStack Documentation
```

Comment:
why we need README.OLD.md here ?

Reply:
We don't, `docsify init` took the root readme and put it here. The readme in `docs` is what shows in the home page of the docs site, I just thought it was better to leave it this way. We can just use `README.OLD.md` instead.